app/models/PreviouslyRegistered.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.i18n.Messages
+import play.api.libs.json._
+import uk.gov.hmrc.govukfrontend.views.Aliases.Hint
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.RadioItem
+import utils.Enumerable
+
+sealed trait PreviouslyRegistered
+
+object PreviouslyRegistered {
+
+  case object YesNotLoggedIn extends WithName("yesNotLoggedIn") with PreviouslyRegistered
```

Comment:
case class names aren't clear for me. I don't know the concept maybe because of this.
```
YesNotLoggedIn
YesStopped
No
```

Reply:
I have renamed them